internal/tags/tags.go

```diff
@@ -0,0 +1,64 @@
+package tags
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/trento-project/trento/internal/consul"
+)
+
+const kvTagsPath string = "trento/v0/tags/%s/%s/"
```

Comment:
We have the rest of the KV paths stored in the `consul/kv.go`. I don't know if we could move it there, for consistency, as the package is already imported